packages/AutoTDD-Exceptions.package/GHAAccessDeniedError.class/class/accessDeniedMessage.st

```diff
@@ -1,4 +1,4 @@
 accessing
```

Comment:
Recategorize to constants along with all other constant Error messages in Exception package